### PR TITLE
fix new rss-sync threshold

### PR DESCRIPTION
**Pull request by schumi2004**
#### Database Migration
NO

#### Description
This should fix commit #428 done for increasing RSS sync threshold #292
Currently the GUI doesn't allow you to use new values, this should resolve it.

### CODE REVIEW

File: src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ protected override bool IsValid(PropertyValidatorContext context)
                 return true;
             }
 
-            if (value >= 10 && value <= 120)
+            if (value >= 60 && value <= 720)
```
Comment (galli-leo) [inline src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs]: Can you change that to 10 again? We still want users without rate limits to go pretty low

File: src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@ namespace NzbDrone.Api.Validation
     public class RssSyncIntervalValidator : PropertyValidator
     {
         public RssSyncIntervalValidator()
-            : base("Must be between 10 and 120 or 0 to disable")
+            : base("Must be between 60 and 720 or 0 to disable")
```
Comment (galli-leo) [inline src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs]: Here too change it to 10.

File: src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs (hunk 1 above)
Comment (schumi2004) [inline src/NzbDrone.Api/Validation/RssSyncIntervalValidator.cs]: That would mean changing commit #428 as well. Will update PR.

**Comment by schumi2004** [comment]
PR updated

**Pull request merged**